src/models/userDao.js

```diff
@@ -0,0 +1,179 @@
+const { AppDataSource } = require("../models/dataSource");
+const { createConnection } = require("typeorm");
+const { throwError } = require("../utils");
+
+// isSubscribed 추가
+const findByKakaoId = async (kakaoId) => {
+  const [result] = await AppDataSource.query(
+    `
+        SELECT id, user_type
+        FROM users
+        WHERE kakao_id = ?
+        `,
+    [kakaoId]
+  );
```

Comment:
```suggestion
// isSubscribed 추가
const findUserByKakaoId = async (kakaoId) => {
  const [result] = await AppDataSource.query(
    `
        SELECT id, user_type
        FROM users
        WHERE kakao_id = ?
        `,
    [kakaoId]
  );
```

- Dao단에서 생성하는 함수의 이름에는 데이터의 이름이 꼭 들어가는 것이 좋습니다. 어떤 자원을 수정하는지, 또는 어떤 자원에 접근하는지 명확히 하기 위해서입니다.